data-reconciliation-app/test/e2e-test/src/api.ts

```diff
@@ -0,0 +1,41 @@
+import { DemoMemberProps, DemoProps } from './index';
+import axios from 'axios';
+import { SummaryRecordProps } from '../../../src/models/summary-record';
+
+export interface ReportItem {
```

Comment:
I realised we are returning a custom response from `/report` and it is not covered by `models`. So, this interface is used for typecasting that response.